src/components/Div.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import Registry from '../utils/Registry';
+import BaseComponent from './BaseComponent';
+
+export default class Div extends BaseComponent {
```

Comment:
I would extend just Component since it's not using any feature of the base component.